src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeChest.java

```diff
@@ -98,7 +98,7 @@ protected ModularUI createUI(EntityPlayer entityPlayer) {
     @Override
     public void update() {
         super.update();
-        if (getOffsetTimer() % ticksPerCycle != 0) return;
+        if (ticksPerCycle != 0 || getOffsetTimer() % ticksPerCycle != 0) return;
```

Comment:
Shouldn't this be `ticksPerCycle == 0`? Same for the Creative tank

Reply:
fixed